.github/workflows/sync-bot.yml

```
@@ -0,0 +1,51 @@
+name: 🤖Sync Bot
+
+
+defaults:
+  run:
+    shell: pwsh
+
+
+on:
+  issues:
+    types: [labeled, unlabeled, assigned, unassigned, milestoned, demilestoned]
+
+
+jobs:
+  sync_bot:
+    name: Sync Bot
+    if: ${{ !github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set Up Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+
+      - name: Run Sync Bot (Issue Change)
+        run: |
+          $scriptUrl = "${{ vars.SCRIPT_BASE_URL }}/${{ vars.CICD_SCRIPTS_VERSION }}/${{ vars.SCRIPT_RELATIVE_DIR_PATH}}/sync-bot-status-check.ts";
+
+          $issueNumber = "${{ github.event.issue.number }}";
+
+          Write-Host "::notice::Project Name: ${{ vars.PROJECT_NAME }}";
+          Write-Host "::notice::Issue: $issueNumber";
+
+          if ($manuallyExecuted -and $issueNumber -eq "0") {
+            Write-Host "::error::The issue or PR number must be a value greater than 0.";
+            exit 1;
+          }
+
+          <# Deno Args:
+            1. Project Name
+            2. Issue Number
+            3. Event Type - set to issue event type
+            4. GitHub token
+          #>
+          deno run `
+            --allow-net `
+            "$scriptUrl" `
+            "${{ vars.PROJECT_NAME }}" `
+            "$issueNumber" `
+            "issue" `
+            "${{ secrets.CICD_REST_API }}";
```

Update token reference:

```suggestion
            "${{ secrets.CICD_TOKEN }}";
```